apps/dashboard/src/lib/components/Course/components/Lesson/Exercise/index.svelte

```diff
@@ -94,6 +94,7 @@
 
     <RoleBasedSecurity allowedRoles={[1, 2]}>
       <ContentSwitcher bind:selectedIndex class="mb-2">
+        <!-- come back here, the numbering is buggy -->
```

Comment:
@tunny17 What does this mean?

Reply:
I noticed when i was adding and deleting the questions on edit mode, the numbering was wrong at some point like it wasn't getting the live update of the number of questions present

Reply:
@tunny17 Please share a video of how this works.